src/zenml/integrations/kaniko/__init__.py

```diff
@@ -25,7 +25,7 @@ class KanikoIntegration(Integration):
     """Definition of the Kaniko integration for ZenML."""
 
     NAME = KANIKO
-    REQUIREMENTS = ["kubernetes==18.20.0"]
+    REQUIREMENTS = []
```

Comment:
no requirements? :D

Reply:
then what does `integration install` even do in that case?

Reply:
Nothing :D